pyemma/coordinates/data/_base/iterable.py

```diff
@@ -172,7 +174,7 @@ def _create_iterator(self, skip=0, chunk=0, stride=1, return_trajindex=True, col
 
     def output_type(self):
         r""" By default transformers return single precision floats. """
-        return np.float32
+        return np.float32()
```

Comment:
why do we need an instance at this point?

Reply:
we don't need it but it is how numpy usually operates i would say:
```python
np.arange(0,10,100).dtype
```
yields an instance rather than the class

Reply:
it is more convenient - I see.